packages/app-store/_utils/getEventTypeAppData.ts

```diff
@@ -28,6 +28,7 @@ export const getEventTypeAppData = <T extends EventTypeAppsList>(
       price: eventType.price,
       // Currency default is "usd" in DB.So, it would also be available always
       currency: eventType.currency,
+      paymentOption: "ON_BOOKING",
```

Comment:
I wasn't sure what to call the two payment scenarios. I wanted to reserve "payment method" if we ever want to accept other payments that were not cards. 

In this PR I want to introduce two options. `ON_BOOKING` is our normal flow where we collect the payment on booking.  The other option `HOLD` will be when we want to collect the payment information but not capture the funds yet.